src/libraries/System.Net.Http/tests/FunctionalTests/SocketsHttpHandlerTest.cs

```diff
@@ -1634,6 +1634,8 @@ public sealed class SocketsHttpHandler_HttpClientHandler_MaxResponseHeadersLengt
     }
 
     [ConditionalClass(typeof(HttpClientHandlerTestBase), nameof(IsQuicSupported))]
+    [ActiveIssue("https://github.com/dotnet/runtime/issues/94507")]
```

Comment:
```suggestion
    [ActiveIssue("https://github.com/dotnet/runtime/issues/91757")]
```

Actually, can you change this to point at #91757?
They're tracking the same pattern - I'll close #94507 as duplicate